templates/web/fixmystreet.com/index.html

```diff
@@ -0,0 +1,20 @@
+[% map_js = PROCESS 'front/javascript.html' %]
```

Comment:
You know, I've just tried it out with `footer-marketing.html` being displayed on the homepage (ie: simply removing the index.html override) and it doesn't actually look that bad:

![screen shot 2016-05-06 at 14 58 25](https://cloud.githubusercontent.com/assets/739624/15075224/20978e46-139b-11e6-80bf-ed585175de96.png)

I'll revert the `index.html` change, and we can revisit later if it looks like there's too much duplication.